geode-connectors/src/main/java/org/apache/geode/connectors/util/internal/MappingCommandUtils.java

```diff
@@ -90,6 +90,7 @@ public static boolean isMappingSynchronous(CacheConfig cacheConfig, RegionConfig
   public static boolean isAccessor(RegionAttributesType attributesType) {
     if (attributesType.getDataPolicy() == RegionAttributesDataPolicy.EMPTY
         || (attributesType.getPartitionAttributes() != null
+            && attributesType.getPartitionAttributes().getLocalMaxMemory() != null
```

Comment:
it will not be null. 

There's code:
      case "PARTITION_PROXY": {
        regionAttributes.setDataPolicy(RegionAttributesDataPolicy.PARTITION);
        regionAttributes.setLocalMaxMemory("0");
        break;
      }